source/common/http/filter_manager.cc

```diff
@@ -916,6 +916,11 @@ void FilterManager::sendLocalReply(
 
   stream_info_.setResponseCodeDetails(details);
 
+  if (Runtime::runtimeFeatureEnabled(
```

Comment:
ugh, on further thought I think rather than do this in the filter manager we shoud do it at call sites, so if someone upstream wants to return that old value they're allowed to.  If remoteDecodeComplete isn't available, you can probably check to see if the last byte of the request is received by checking to see if the timing information in streaminfo is set.  Sorry for missing this on first pass.

Reply:
Yeah, I also think about this, what if the caller does want to return 408 for some unpredictable reason.

I initially wanted to add more "modify" policy as was done in `void FilterManager::sendLocalReply(... const std::function<void(ResponseHeaderMap& headers)>& modify_headers,`, but it's probably too early to add. 

Thank you for the tip on remoteDecodeComplete not available. I will update